rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/LocalInfrastructure.java

```diff
@@ -174,11 +178,12 @@ private void acquireNodeImpl() {
                     String out = Tools.join(processExecutor.getOutput(), "\n");
                     String err = Tools.join(processExecutor.getErrorOutput(), "\n");
                     message += "stdout: " + out + lf + "stderr: " + err;
-                    this.declareDeployingNodeLost(depNodeURL, message);
+                    for (int nodeIndex = 0; nodeIndex < handledNodes.get(); nodeIndex++) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS134)